src/AddToCalendar.tsx

```diff
@@ -37,12 +37,17 @@ const useOpenState = (initialOpen: boolean): [boolean, OpenStateToggle] => {
   useEffect(
     () => {
       if (open) {
-        const onClose = () => setOpen(false);
-        document.addEventListener("click", onClose);
-
-        return () => document.removeEventListener("click", onClose);
+        const onClose = () => {
```

Comment:
I couldn't get the locally served file working for me, even when I used your tag 1.1.2

You can ignore this but keen to know if the original actually works for you ?